modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -346,6 +394,17 @@ public void start(@Nullable String cfg) {
         }
     }
 
+    /**
+     * Checks the node up to date by metadata.
+     *
+     * @param metastorageRevision Metastorage revision.
+     * @param appliedRevision Last applied node revision.
+     * @return True when the applied revision is greater enough to node recovery complete, false otherwise.
+     */
+    private boolean isMetadataUpToDate(long metastorageRevision, long appliedRevision) {
+        return metastorageRevision - 100 < appliedRevision;
```

Comment:
maybe we need IgniteSystemProperty instead of `100`

Reply:
We have no this practice in Ignite 3.
We may use a configuration (but now we also have no something like this) to it.
I think TODO is needed here, but nothing more required...